tools.md

```diff
@@ -14,16 +14,52 @@ sidenav:
 ## USWDS Component Library in Figma
 
 ## Figma tokens plug-in
+#### What is Figma Tokens? ####
```

Comment:
In this context, these should be marked up as H3s (nested headings should follow the number in the sequence). But since we're going to break this out into a separate page, I would mark them up as H2s, and the make the page title (Figma Tokens plug-in) an H1.